realm/realm-library/src/main/cpp/io_realm_internal_TableQuery.cpp

```diff
@@ -19,9 +19,11 @@
 #include <shared_realm.hpp>
 #include <object_store.hpp>
 #include "util.hpp"
+#include "jni_util/java_local_ref.hpp"
 #include "io_realm_internal_TableQuery.h"
 
 using namespace realm;
+using namespace realm::jni_util;
```

Comment:
I am trying to stick with:
* always use full namespace in the header files -- I think that is why you say "you in other files add using but still make calls with jni_util::" 
* always use `using namespace` in the source files since it seems to be an convention for OS and core.

I really like to have a rule to follow, when shall we use `using namespace` and when shall we not?